connectors/kafka/pom.xml

```diff
@@ -42,7 +42,12 @@
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka_2.10</artifactId>
       <version>0.8.2.2</version>
-      <scope>provided</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>*</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
I just verified that each of these non-edgent deps need the non-transitive specification (not just kafka_2.10).  Without it, copy-dependencies (and uber jar) pull in unwanted things relative to the gradle/ant build:

Only in DEPS-WITH-KAFKA-TOO-MUCH/: jline-0.9.94.jar
Only in DEPS-WITH-KAFKA-TOO-MUCH/: lz4-1.2.0.jar
Only in DEPS-WITH-KAFKA-TOO-MUCH/: netty-3.7.0.Final.jar
Only in DEPS-WITH-KAFKA-TOO-MUCH/: slf4j-log4j12-1.6.1.jar
Only in DEPS-WITH-KAFKA-TOO-MUCH/: snappy-java-1.1.1.7.jar